HealthMonitoring.SelfHost/Content/Scripts/homeController.js

```diff
@@ -23,29 +26,68 @@
         var configFrequency = getConfigUpdatingFrequency($location);
 
         $scope.displayHint = function (evnt) {
-            if ($scope.statLegend == null) {
+            if ($scope.statLegend === null) {
                 $scope.statLegend = {
                     status: evnt.target.innerHTML,
                     description: statusLegend[evnt.target.innerHTML],
                     x: evnt.clientX + 5,
                     y: evnt.clientY + 5
                 };
             }
-        }
+        };
+
+        $scope.getGroupColour = function (group) {
```

Comment:
Also. Do this implementation correct? 
f.e. if we have such thing:
group.TimedOut = 3
and group.Faulty  = 6 
in this case we will return 'TimedOut ' 
is this that behavior we want?
maybe it will be more correct if we will show status that has maximum count or set some priority (faulty will be max priority status(f.e.) so if we have a least one faulty endpoint so whole group will be faulty)